modules/javafx.graphics/src/main/java/com/sun/glass/ui/Accessible.java

```diff
@@ -162,36 +162,30 @@ private class GetAttribute implements PrivilegedAction<Object> {
 
     private GetAttribute getAttribute = new GetAttribute();
 
-    @SuppressWarnings("removal")
     public Object getAttribute(AccessibleAttribute attribute, Object... parameters) {
-        AccessControlContext acc = getAccessControlContext();
-        if (acc == null) return null;
         return QuantumToolkit.runWithoutRenderLock(() -> {
             getAttribute.attribute = attribute;
             getAttribute.parameters = parameters;
-            return AccessController.doPrivileged(getAttribute, acc);
+            return getAttribute;
```

Comment:
This needs to be `getAttribute.get()` to be equivalent to the previous code.

Reply:
Alternatively, the following would work and be simpler, if you prefer:

```
    public Object getAttribute(AccessibleAttribute attribute, Object... parameters) {
        getAttribute.attribute = attribute;
        getAttribute.parameters = parameters;
        return QuantumToolkit.runWithoutRenderLock(getAttribute);
    }
```

Reply:
Thanks for noticing this Kevin. This was root cause for ClassCastException seen when running HelloTextArea with VoiceOver enabled. I have verified that with updated code we don't see any exceptions.